features/lwipstack/lwip/src/apps/altcp_tls/altcp_tls_mbedtls.c

```diff
@@ -734,7 +761,14 @@ altcp_tls_create_config(int is_server, int have_cert, int have_pkey, int have_ca
   }
   mbedtls_ssl_conf_authmode(&conf->conf, MBEDTLS_SSL_VERIFY_OPTIONAL);
 
+#if !defined(MBEDTLS_SSL_CONF_RNG)
+#if defined(MBEDTLS_CTR_DRBG_C)
   mbedtls_ssl_conf_rng(&conf->conf, mbedtls_ctr_drbg_random, &conf->ctr_drbg);
+#elif defined(MBEDTLS_HMAC_DRBG_C)
+  mbedtls_ssl_conf_rng(&conf->conf, mbedtls_hmac_drbg_random, &conf->ctr_drbg);
+#endif
```

Comment:
No, we don't want to pollute code with dead code

Reply:
So only one of those 2 conditions is always guaranteed? If that is the case then the #elif should be a #else ? Otherwise you have unexpected code execution....

Reply:
Same point as [above](https://github.com/ARMmbed/mbed-os/pull/12479#discussion_r400106285).

Reply:
Fixed